packages/connect/src/device/DeviceCommands.ts

```diff
@@ -304,27 +293,9 @@ export class DeviceCommands {
         });
 
         const res = await this.callPromise;
-
         this.callPromise = undefined;
-        if (!res.success) {
-            logger.warn(
-                'Received error',
-                res.error,
-                // res.message is not propagated to higher levels, only logged here. webusb/node-bridge may return message with additional information
-                res.message,
-            );
-            throw new Error(res.error);
```

Comment:
Maybe I'm missing the point, but why don't we just do the `throw new Error(res.error, { cause: 'transport-error' })` thing here, instead of all the other changes? It would propagate upwards anyway, right?